templates/blog/index.html

```diff
@@ -0,0 +1,29 @@
+{% extends "base.html" %}
+{% block content %}
+  <ul>
+    {% for post in post_list %}
+      <li><a href="{% url 'blog:post-detail' pk=post.pk %}">{{ post.title }}</a></li>
+      <p>{{ post.created_time }}</p>
+      <p>{{ post.owner }}</p>
+      <p>{{ post.content }}</p>
+      <p>commentaries: {{ post.commentaries_count }}</p>
+    {% endfor %}
+  </ul>
+  {% if is_paginated %}
+  <ul class="pagination">
+    {% if page_obj.has_previous %}
+      <li class="page-item">
+        <a href="?page= {{ page_obj.previous_page_number }}" class="page-link">prev</a>

```

Comment:
Remove the space after '=' in the pagination link to ensure the query parameter is correctly formatted. It should be `?page={{ page_obj.previous_page_number }}`.